test/jdk/java/awt/dnd/RejectDragDropActionTest.java

```diff
@@ -97,11 +97,11 @@ public static void main(String[] args) throws InterruptedException,
 
             robot.mouseMove(startPoint.x, startPoint.y);
             robot.mousePress(InputEvent.BUTTON1_DOWN_MASK);
-            for (Point p = new Point(startPoint); !p.equals(endPoint);
+            for (Point p = new Point(startPoint);
+                 !p.equals(endPoint) && !incorrectActionDetected;
                  p.translate(sign(endPoint.x - p.x),
                          sign(endPoint.y - p.y))) {
```

Comment:
```suggestion
                 p.translate(sign(endPoint.x - p.x),
                             sign(endPoint.y - p.y))) {
```
I'm probably the only one who prefers aligning wrapped lines… Doesn't it look better?

Reply:
I agree, it does look neater. Unfortunately, my IDE doesn't default this (which makes sense) so I have to remember to do this when applicable. Thanks.

Reply:
I configured IDE to do it by default. Yet it doesn't make sense in all the cases.